gamemodes/sss/core/weapon/ammunition.pwn

```diff
@@ -117,7 +117,7 @@ hook OnItemNameRender(Item:itemid, ItemType:itemtype)
 
 	SetItemNameExtra(itemid, str);
 
-	return Y_HOOKS_CONTINUE_RETURN_0;
+	return 1;
```

Comment:
What exactly does this fix? I see the changes from constants to literals, that's fine but why the change in hook chain logic?